pkg/lib/verifier/rulebuilder_intel_tpm20.go

```diff
@@ -314,29 +310,22 @@ func (builder *ruleBuilderIntelTpm20) getPlatformPcrsFromHardwareMeta() ([]types
 		return nil, errors.New("The flavor's Feature information is not present")
 	}
 
-	if feature.CBNT != nil {
-		if feature.CBNT.Enabled {
-			if feature.CBNT.Profile == "BTGP5" {
-				pcrs = append(pcrs, types.PCR7)
-			}
+	if feature.CBNT.Enabled {
+		if feature.CBNT.Meta.Profile == "BTGP5" {
+			pcrs = append(pcrs, types.PCR7)
 		}
 	}
 
-	if feature.SUEFI != nil {
-		if feature.SUEFI.Enabled {
-			suefiPcrs := []types.PcrIndex{
-				types.PCR1,
-				types.PCR2,
-				types.PCR3,
-				types.PCR4,
-				types.PCR5,
-				types.PCR6,
-				types.PCR7,
-			}
-
-			pcrs = append(pcrs, suefiPcrs...)
-		}
+	suefiPcrs := []types.PcrIndex{
```

Comment:
these rules must be part of the template I guess

Reply:
The changes related to flavor verification are in https://github.com/pkumarex/intel-secl/pull/5.

Reply:
why static rules are defined it should be part of template right?